src/exojax/spec/tau1height.py

```diff
@@ -0,0 +1,24 @@
+"""Extract tau=1 height pressure at different wavelengths"""
+import numpy as np
+from scipy import interpolate
+
+def get_P_at_tauone(dtau, Parr, tauextracted=1.0):
```

Comment:
`get_P_at_tauone` -> `extract_pressure_at_tauone`? (from the comment)
or simply `pressure_at_given_opacity`? any good idea?

Reply:
Thanks for the suggestion! Okay, I'll choose `pressure_at_given_opacity`.